src/main/java/com/amazon/opendistroforelasticsearch/sql/parser/WhereParser.java

```diff
@@ -576,8 +576,10 @@ private SQLMethodInvokeExpr parseSQLBinaryOpExprWhoIsConditionInWhere(SQLBinaryO
 
         String operator = soExpr.getOperator().getName();
 
-        if (operator.equals("=")) {
+        if (operator.equals("=") || operator.equals("IS")) {
             operator = "==";
+        } else if (operator.equals("IS NOT")) {
```

Comment:
do we maintain enums for these string literals ? 
eg. IS / IS NOT
@davidcui-amzn @dai-chen @penghuo

Reply:
> do we maintain enums for these string literals ?
> eg. IS / IS NOT
> @davidcui-amzn @dai-chen @penghuo

There is enum we added in type system in new ANTLR parser: https://github.com/opendistro-for-elasticsearch/sql/blob/master/src/main/java/com/amazon/opendistroforelasticsearch/sql/antlr/semantic/types/operator/ComparisonOperator.java#L37. But I guess nothing in the old codebase.

Reply:
@davidcui-amzn please check if case is ignored or not, ex `is` and `is not`.